src/Symfony/Component/Console/Application.php

```diff
@@ -117,7 +117,7 @@ public function setCommandLoader(CommandLoaderInterface $commandLoader)
 
     public function getSignalRegistry(): SignalRegistry
     {
-        if (!$this->signalRegistry) {
+        if (!isset($this->signalRegistry)) {
```

Comment:
What about making it nullable instead and init it with `null` by default?

Reply:
It's the same for me, I'll change